src/database/models.py

```diff
@@ -654,8 +651,10 @@ class Team(models.Model):
   members = models.ManyToManyField(UserProfile, related_name='team_members',
                                    blank=True)
   
-  # may change this from ForeignKey to ManyToManyField to allow team to span communities
-  community = models.ForeignKey(Community, on_delete=models.CASCADE)
+  # change this from ForeignKey to ManyToManyField to allow team to span communities
+  # rename community to primary_community - this is the one whose cadmin can add/delete other communities, and which is unique with name
+  communities = models.ManyToManyField(Community, related_name='team_communities', blank=True)
```

Comment:
rename `related_name` from `team_communities` to `community_teams` and `team_community` to `primary_community_teams`